src/main/java/calculator/Numbers.java

```diff
@@ -0,0 +1,35 @@
+package calculator;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Numbers {
+    private final List<Number> values;
+
+    private Numbers(final List<Number> values) {
+        this.values = values;
+    }
+
+    public static Numbers from(final String[] values) {
+        List<Number> numbers = Arrays.stream(values)
+                .map(Number::from)
+                .collect(Collectors.toList());
+
+        return new Numbers(numbers);
+    }
+
+    public int sum() {
+        return values.stream()
+                .mapToInt(Number::getValue)
+                .sum();
+    }
+
+    public List<Number> getValues() {
+        return values;
+    }
```

Comment:
value를 직접 반환하는 것 보다는, 불변 객체를 반환하는 방향도 좋을 것 같습니다.

```suggestion
    public List<Number> getValues() {
	return Collections.unmodifiableList(values);
    }
```

참고 자료
[https://tecoble.techcourse.co.kr/post/2021-04-26-defensive-copy-vs-unmodifiable/](https://tecoble.techcourse.co.kr/post/2021-04-26-defensive-copy-vs-unmodifiable/)

Reply:
피드백 주셔서 감사합니다. 추후 미션 진행시 참고하도록 하겠습니다!